package.json

```diff
@@ -42,6 +42,7 @@
   "devDependencies": {
     "danger": "^3.7.14",
     "eslint": "^5.0.0",
+    "eslint-plugin-react-hooks": "^1.0.0",
```

Comment:
Shouldn't this be a dependency?

Reply:
Updated: https://github.com/cerner/eslint-config-terra/pull/24/commits/f9ab57e5999b6be7ee012505b439be9afae32197